core/impl-base/src/test/java/org/jboss/arquillian/core/impl/loadable/ServiceRegistryLoaderTestCase.java

```diff
@@ -44,7 +46,7 @@
    @Test
    public void shouldBeAbleToLoadAll() throws Exception
    {
-      ServiceRegistry registry = new ServiceRegistry(injector.get());
+      ServiceRegistry registry = new ServiceRegistry(injector.get(), new LinkedHashMap<Class<?>, Set<Class<?>>>());
```

Comment:
I see there is also ServiceRegistryTestCase - how is it different from this test class which interacts with the same component?